### ISSUE TITLE
[Bug]: GameOverDialog logo images are too large

### ISSUE THREAD

**Issue by itsalaidbacklife** [cuttle-cards/cuttle#327]
### Bug Summary

When the game ends, each player sees a dialog with an image that conveys whether they won or lost. Those images are currently too large on mobile

### Detailed Description

See image (iphone-8, landscape)
![image](https://user-images.githubusercontent.com/6520704/217013499-373ea1b6-ffe3-40b4-82fc-26b90055aa65.png)

The image is too big to see both the heading and the button at the bottom. It should be ~1/2 the size

### Steps to Reproduce

1.
2.
3.

**Comment by seriouslysean** [comment]
@itsalaidbacklife pretty sure we recently fixed this, so we need them smaller still? If so might be worth adding a recommended size per breakpoint in this issue.

**Comment by itsalaidbacklife** [comment]
I believe this is still an issue on mobile devices in landscape mode. Agreed that we should spec the requirements more closely

**Comment by TommySchmid** [comment]
Last comment was in May so I decided to take a look. Easy one to get back into it.

I took a game over spec and added `cy.viewport('iphone-8', 'landscape');` - screenshot is below. Looks like this issue can be closed @itsalaidbacklife 

Let me know if anything else is needed to close this one.

![iphone 8 landscape](https://github.com/cuttle-cards/cuttle/assets/44646343/3d599848-deb5-4f5b-8101-13aba62d9032)
